Controller/AsseticController.php

```diff
@@ -82,7 +82,7 @@ public function render($name, $pos = null)
             $response->setETag(md5(serialize($formula)));
         }
 
-        if ($response->isNotModified($this->request)) {
+        if ($response->isNotModified($this->requestStack->getMasterRequest())) {
```

Comment:
(apply also to line [107](https://github.com/symfony/AsseticBundle/pull/337/files#diff-9be19f19162e0c6014663b2eb2575bdbR107))

1) not sure about using `requestStack->getMasterRequest()` or `requestStack->getCurrentRequest()`

2) `requestStack->getMasterRequest()` can return null, thinking about how to handle it